src/kde_analysis/eval_m1m2chieff_poiss_weights.py

```diff
@@ -187,16 +441,25 @@ def get_rate_m_chieff2D(m1_query, m2_query, Rate):
     group = hdf[iter_name]
     if 'bootstrap_weights' in group:
         boots_weighted = True
-        poisson_weights = group['bootstrap_weights'][:]
-        assert min(poisson_weights) > 0, "Some bootstrap weights are non-positive!"
-        Nboots = poisson_weights.sum()  # Number of events in bootstrap
+        weights = group['bootstrap_weights'][:]
+        assert min(weights) > 0, "Some bootstrap weights are non-positive!"
```

Comment:
This line is commented out in https://github.com/jam-sadiq/MultidimensionalIterativeKDE/commit/1ee2b7610116826d6cee443dcb4e229b750e08ce - please also comment it here (we want to allow zero weights!)